robottelo/hosts.py

```diff
@@ -1515,3 +1533,82 @@ def install_cockpit(self):
                 f'Error during cockpit installation, installation output: {cmd_result.stdout}'
             )
         self.add_rex_key(self)
+
+    def wait_for_tasks(
```

Comment:
@Gauravtalreja1 I think this helper method is more suited in previous location(robottelo.api.utils).  @JacobCallahan @jyejare any thoughts?

Reply:
Okay, moved it to capsule_mixin under CapsuleInfo class